core/service/celery/celery.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from celery import Celery
+from kombu import Exchange, Queue
+
+
+def task_queue_config(cm_name: str) -> dict[str, Any]:
+    high_prefixed = f"{cm_name}:high"
+    default_prefixed = f"{cm_name}:default"
```

Comment:
Prefix our task queues with the `cm_name`, so that hopefully we are able to have a redis instance per-shard when we roll this out. This has only been lightly tested. It seems like it will work because Celery creates redis keys for each queue, but I need to do a lot more testing of this part.

Reply:
Changed this to use a broker option instead.